src/wallet/rpcwallet.cpp

```diff
@@ -3741,25 +3741,18 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
                         {RPCResult::Type::STR_HEX, "pubkey", /* optional */ true, "The hex value of the raw public key for single-key addresses (possibly embedded in P2SH or P2WSH)."},
                         {RPCResult::Type::OBJ, "embedded", /* optional */ true, "Information about the address embedded in P2SH or P2WSH, if relevant and known.",
                         {
-                            {RPCResult::Type::ELISION, "", "Includes all\n"
```

Comment:
Unrelated, but if you are reformatting these lines could you clean up [line 1544](https://github.com/bitcoin/bitcoin/blob/bc01f7ae0538d3c647ce8dfbc29f7914d5df3fbb/src/wallet/rpcwallet.cpp#L1544)?

Reply:
Good idea. Will touch up in the next PR to not invalidate review.

Reply:
Done in #20220.

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/20220/files#diff-63c7710d72381cffe692175dd70e47f1891bf75c3e09249ed826f9df3917d02dR3791

Reply:
Decided to do it in another PR or code a general solution to line wrapping in the RPC help rather than manual adjustments.